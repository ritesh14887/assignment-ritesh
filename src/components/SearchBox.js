

import React, { Component } from 'react';

class SearchBox extends Component {
    constructor() {
        super();
    }

    render() {
        return (
            <div className="search-wrap">
                <h3 className="form-legend">Search By name</h3>
                <form className="search-box">
                    <input type="search" />
                    <button>Search</button>
                </form>
            </div>
        );
    }
}

export default SearchBox;