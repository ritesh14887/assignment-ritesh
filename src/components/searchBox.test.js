import React from 'react';
import renderer from 'react-test-renderer';
import SearchBox from './SearchBox';

test('Pagination renders correctly', () => {
  const pageData = {
    pages: 2,
  };

  const component = renderer.create(
    <SearchBox />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
