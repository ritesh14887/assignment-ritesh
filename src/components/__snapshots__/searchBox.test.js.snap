// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination renders correctly 1`] = `
<div
  className="search-wrap"
>
  <form
    className="search-box"
  >
    <input
      onChange={[Function]}
      type="text"
    />
    <button
      onClick={[Function]}
      type="button"
    >
      Search
    </button>
  </form>
</div>
`;
